src/sentry/apidocs/api_ownership_allowlist_dont_modify.py

```diff
@@ -0,0 +1,491 @@
+"""
+    This list is tracking old apis that we couldn't find owners for.
+    The goal is to eventually find owners for all and shrink this list.
+    DO NOT ADD ANY NEW APIS
```

Comment:
you'll want to add yourself (or a team) to CODEOWNERS for this file -- otherwise it will regress unfortunately :/

Reply:
Yeah, I add this file to our github review so it requires review from us. Something like this: https://github.com/orgs/community/discussions/49633 
I don't have the right permissions so yeah, in progress.

Reply:
The file is already added to codeowners. Seems like we can't do above in our repo so I'll look into other ways.